data/campaigns/The_South_Guard/scenarios/07a_Into_the_Depths.cfg

```diff
@@ -232,6 +233,28 @@
             facing=sw
             ai_special=guardian
         [/unit]
+
+        ## guardians in M'Brin's keep
+#ifdef EASY
```

Comment:
This block can be better expressed with `#ifndef EASY`.

No need to make a new PR for this, it's just enough to add a new commit and push your branch again.